omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs1_9/ConceptSearchHandler1_9.java

```diff
@@ -48,9 +50,9 @@ public class ConceptSearchHandler1_9 implements SearchHandler {
 	
 	private final SearchConfig searchConfig = new SearchConfig("byTerm", RestConstants.VERSION_1 + "/concept",
 			Collections.singletonList("1.9.* - 9.*"),
-	        new SearchQuery.Builder(
-	                "Allows you to find concepts which map to term, uuid of term given as input").withRequiredParameters(
-	            "term").build());
+			Arrays.asList(new SearchQuery.Builder("Allows you to find concepts which map to term, uuid of term given as input")
+					.withRequiredParameters("term").build(), new SearchQuery.Builder("Allows you to find concepts by uuid and mapping")
+					.withRequiredParameters("uuid").build()));
```

Comment:
Hey @dkayiwa I have added a new search query in the existing endpoint. What have done is to introduce two search variables conceptUuid & mapping, and restricted them to be passed together when  requesting concepts. 

Is what have done ok?